docs/blog/2018-10-04-journey-to-the-content-mesh/index.md

```diff
@@ -0,0 +1,62 @@
+# Delivering Modern Website Experiences: The Journey to a Content Mesh
+
+The CMS was born to make content sites _possible_. Now, it’s being reimagined to make them _incredible_.
+
+In the early 2000s, publishing on the web was difficult. The CMS emerged as a single application to store content, build sites, and deliver them to users.
+
+Over time, tools in each category got better and more specialized, while the quality of monolithic enterprise CMS applications like Adobe Experience Manager and Sitecore has stayed roughly the same.
```

Comment:
It isn't clear what you mean by "category".

Also, few would agree that the quality of AEM and Sitecore has stayed the same. If nothing else, they've added lots of features.

Reply:
The story needs fleshed out perhaps along the lines of a classic disruption innovation story — that as CMSs got more complicated, simpler tools that only did part of the CMS emerged that allowed people with specialized needs to move faster & w/ more simplicity

Reply:
The paragraphs below do a great job of explaining that — just feels like needs more connecting tissue

Reply:
Great point -- added some more "connective tissue" foreshadowing the net couple of paragraphs. Of course they've added features -- and if you're adding more features as client requirements grow at roughly the same speed, your quality remains constant. If client requirements are growing faster, you can add features while simultaneously declining in quality! :)